lib/autoInject.js

```diff
@@ -4,10 +4,10 @@ import arrayMap from 'lodash/_arrayMap';
 import clone from 'lodash/_copyArray';
 import isArray from 'lodash/isArray';
 
-var argsRegex =  /^function\s*[^\(]*\(\s*([^\)]*)\)/m;
+var argsRegex = /^(function[^\(]*)?\(?\s*([^\)=]*)/m;
 
 function parseParams(func) {
-    return  func.toString().match(argsRegex)[1].split(/\s*\,\s*/);
+    return func.toString().match(argsRegex)[2].trim().split(/\s*\,\s*/);
```

Comment:
A good read on coercing functions to strings:
http://perfectionkills.com/state-of-function-decompilation-in-javascript/